docs/cloud/azureai/runtime-change-log.md

```diff
@@ -14,6 +14,18 @@ You can check the runtime image version from the flow execution log:
 
 ## Change log
 Default runtime image is continuously updated, and here we record the new features and fixed bugs of each image version.
+
+### 20240228.v3
+
+#### New features
+- NA
+
+#### Bugs fixed
+- Fix bug when collect package tools
+- Update expected error message of 'test_completion_with_chat_model'
+- Fix naming error of legacy 'OpenSourceLLMOnlineEndpointError'
```

Comment:
Better to make them easier to understand for customer. You can use chatgpt to do that.
Also, there should be a "." at the end of each bug.